concourse/steps/release.py

```diff
@@ -678,12 +680,12 @@ def apply(self):
         create_release_step_output = self.context().step_output('Create Release')
         release_tag = create_release_step_output['release_tag_name']
 
+        component_descriptor_v2 = cm.ComponentDescriptor.from_dict(
```

Comment:
note to self: we should definitely add an option to parse component-descriptor from file(-like object)